ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -67,8 +67,6 @@ tests:
 - as: e2e-metal-assisted
   steps:
     cluster_profile: packet
-    dependencies:
-      ASSISTED_AGENT_IMAGE: pipeline:assisted-installer-agent
```

Comment:
why this is not needed anymore?

Reply:
Since the dependencies of the step isn't specifying where it comes from. Just the expected ImageStream tag.
https://github.com/openshift/release/blob/9f4d8ba279694499297a4bde144bf6f8f7e9fc9a/ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-ref.yaml#L9-L17
On this configuration file - the pipeline ImageStream is the only image available, it would always be taken under the same name.